test/utils/attach-policy-to-test-role.ts

```diff
@@ -4,10 +4,10 @@ import type { Stack } from "@aws-cdk/core";
 import { Tags } from "@aws-cdk/core";
 
 // IAM Policies need to be attached to a role, group or user to be created in a stack
-export const attachPolicyToTestRole = (stack: Stack, policy: Policy, app: string = "testing"): void => {
-  const role = new Role(stack, "TestRole", {
+export const attachPolicyToTestRole = (stack: Stack, policy: Policy, id: string = "TestRole"): void => {
```

Comment:
The default value of the `app` param was never overridden, so (safely) removed it.